### PR TITLE
Addressing building errors and grpc update

### LINKED ISSUE #16: Error go build
```
go build -o bin/tcd -installsuffix cgo
# github.com/kinvolk/tcd/api
../.go_workspace/src/github.com/kinvolk/tcd/api/service.pb.go:184: cannot use _TcdService_InstallMethod_Handler (type func(interface {}, context.Context, func(interface {}) error) (interface {}, error)) as type grpc.methodHandler in field value
../.go_workspace/src/github.com/kinvolk/tcd/api/service.pb.go:188: cannot use _TcdService_ConfigureIngressMethod_Handler (type func(interface {}, context.Context, func(interface {}) error) (interface {}, error)) as type grpc.methodHandler in field value
../.go_workspace/src/github.com/kinvolk/tcd/api/service.pb.go:192: cannot use _TcdService_ConfigureEgressMethod_Handler (type func(interface {}, context.Context, func(interface {}) error) (interface {}, error)) as type grpc.methodHandler in field value

go build -o bin/tcd -installsuffix cgo returned exit code 2

Action failed: go build -o bin/tcd -installsuffix cgo
```

### PR DESCRIPTION
**Pull request by alepuccetti**
This fixes: #16

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -1,5 +1,30 @@
-#!/bin/bash -e
+#!/usr/bin/env bash
+
+# This script is a modification of the build for docke2aci
```
Comment (alban) [inline build.sh]: docke2aci -> docker2aci

Comment (alban) [inline build.sh]: @iaguis should it be mentioned where this script comes from?

File: build.sh [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
+if [ ! -h gopath/src/${REPO_PATH} ]; then
+    mkdir -p gopath/src/${ORG_PATH}
+    ln -s ../../../.. gopath/src/${REPO_PATH} || exit 255
+fi
+
+export GOBIN=${PWD}/bin
+export GOPATH=${PWD}/gopath:${PWD}/Godeps/_workspace
+
+eval $(go env)
+go get -d
```
Comment (alban) [inline build.sh]: About the "go get -d": please add a comment to say this needs to be removed after this is vendored, to be sure not to forget.

File: build.sh [hunk 3]
```diff
@@ -1,5 +1,30 @@
...
+
+echo "Building tcd..."
+CGO_ENABLE=0 go build -o $GOBIN/tcd -installsuffix cgo
+
+
 
 # ~/programs/protoc/protoc -I api api/service.proto --go_out=plugins=grpc:api
-CGO_ENABLED=0 go build -o bin/tcd -installsuffix cgo
+#CGO_ENABLED=0 go build -o bin/tcd -installsuffix cgo
```
Comment (alban) [inline build.sh]: No need to keep the "go build" line commented, since you have the new version above. You can remove it

File: circle.yml [hunk 4]
```diff
@@ -6,10 +6,14 @@ machine:
...
-    - go build -o bin/tcd -installsuffix cgo
+    #- pwd
+    #- echo $GOPATH
+    #- git rev-parse HEAD
+    - ./build.sh
+    #- go get -d
+    #- go build -o bin/tcd -installsuffix cgo
     #- go build -o bin/tcdclient client/client.go
-    - docker build -t kinvolk/tcd .
+    #- docker build -t kinvolk/tcd .
```
Comment (alban) [inline circle.yml]: You can remove all the leftover comments now.

**Comment by alban** [comment]
Reviewed. You can squash this PR into one commit for the next review round.

File: build.sh (hunk 1 above)
Comment (iaguis) [inline build.sh]: I don't think there's a need to do that

**Comment by alepuccetti** [comment]
@alban I hope I did the squashing correctly

**Comment by alban** [comment]
LGTM

**Pull request merged**